packages/connectkit/src/utils/getPaymentInfo.tsx

```diff
@@ -179,6 +185,13 @@ export function getPaymentInfo(log: (...args: any[]) => void) {
     [daimoPayOrder],
   );
 
+  const onSuccess = ({ txHash, txURL }: { txHash: string; txURL?: string }) => {
```

Comment:
fine for now but i’ve tried to kept only business logic in this file (ie no ui related stuff) so i would’ve put this in the `Confirmation` `/pages` code in future